tests/DimExtensionsTests.cs

```diff
@@ -6,7 +6,7 @@ namespace UnitTests;
 [TestOf( typeof( DimExtensions ) )]
 [Category( "Core" )]
 [Category( "Terminal.Gui Extensions" )]
-[Parallelizable( ParallelScope.Children )]
+[NonParallelizable]
```

Comment:
Instead using `[NonParallelizable]` attribute everywhere, I think it's possible prevent parallel execution via .`runsettings`:

```xml
  <NUnit>
    <DefaultTestCaseParallelization>false</DefaultTestCaseParallelization>
  </NUnit>

```

Reply:
Yeah you are right - I think this is the default so I could probably just have deleted the attributes.